apps/frontend/src/libs/modules/store/store.module.ts

```diff
@@ -7,9 +7,12 @@ import { configureStore } from "@reduxjs/toolkit";
 
 import { AppEnvironment } from "~/libs/enums/enums.js";
 import { type Config } from "~/libs/modules/config/config.js";
+import { notifier as toastNotifier } from "~/libs/modules/toast-notifier/toast-notifier.js";
```

Comment:
the exported module itself should be named toastNotifier
```suggestion
import { toastNotifier } from "~/libs/modules/toast-notifier/toast-notifier.js";
```